test/e2e/minscale_readiness_test.go

```diff
@@ -150,3 +158,19 @@ func waitForDesiredScale(t *testing.T, clients *test.Clients, deploymentName str
 		return cond(*deployment.Spec.Replicas), nil
 	})
 }
+
+func waitForDesiredEndpoints(t *testing.T, clients *test.Clients, endpointName string, cond func(int32) bool) error {
+	endpoints := clients.KubeClient.Kube.CoreV1().Endpoints(test.ServingNamespace)
+
+	return wait.PollImmediate(time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
```suggestion
	return wait.PollImmediate(time.Second, 1*time.Minute, func() (bool, error) {
```

To stay the same as above?